tests/testImageDecorrelation.py

```diff
@@ -245,19 +265,19 @@ def _testImages(self):
 
         # Expected (ideal) variance of difference image
         expected_var = self.svar + self.tvar
-        print('Expected variance:', expected_var)
+        print('EXPECTED VARIANCE:', expected_var)
```

Comment:
"The stack currently does something" is not normally a good justification, considering a lot of the cruft in there.

The thing I'm worried about is when someone else is working on this code, what are they supposed to do with that printed number? If you know what the rough value should be, maybe an `assertAlmostEqual(expected_var, SOMENUMBER)` is in order, as a sanity check?